lib/GPUArraysCore/src/GPUArraysCore.jl

```diff
@@ -6,7 +6,7 @@ using Adapt
 ## essential types
 
 export AbstractGPUArray, AbstractGPUVector, AbstractGPUMatrix, AbstractGPUVecOrMat,
-       WrappedGPUArray, AnyGPUArray, AbstractGPUArrayStyle
+       WrappedGPUArray, AnyGPUArray, AbstractGPUArrayStyle, AnyGPUArray, AnyGPUMatrix
```

Comment:
Maybe export AnyGPUVector too?

```suggestion
       WrappedGPUArray, AnyGPUArray, AbstractGPUArrayStyle,
       AnyGPUArray, AnyGPUVector, AnyGPUMatrix
```